docs/3.0.0-beta.x/guides/services.md

```diff
@@ -1,18 +1,165 @@
 # Services
 
-See the [services concept](../concepts/concepts.md#services) for details.
+See the [services concept](../concepts/concepts.md#services) for an overview.
 
-## How to create a service?
+## Core services
 
-There is two ways to create a service.
- - Using the CLI `strapi generate:service user`. Read the [CLI documentation](../cli/CLI.md) for more information.
- - Manually create a JavaScript file named `User.js` in `./api/**/services/`.
+When you create a new Content type or a new model. You will see a new empty service has been created. It is because Strapi builds a generic service for your models by default and allows you to override and extend it in the generated files.
+
+### Extending a Model Service
+
+Here are the core methods (and their current implementation).
+You can simply copy and paste this code in your own service file to customize the methods.
+
+Youc an read about `strapi.query` calls [here](./queries.md)
+
+::: warning
+In the following example we will consider your controller, service and model is named `product`
```

Comment:
In the following example, your controller, service and model are named `product`.